frontend/src/pages/Register.vue

```diff
@@ -240,20 +264,43 @@ export default {
           },
         })
         .then(() => {
-          this.$router.push('/thx/register')
+          // Wolle: this.$router.push('/thx/register')
+          this.showPageMessage = true
+          this.success = true
         })
         .catch((error) => {
+          this.showPageMessage = true
+          this.success = false
           this.showError = true
-          this.messageError = error.message
+          switch (errorMessageRemoveGraphQl(error.message)) {
+            case ERRORS.ERR_USER_ALREADY_EXISTS:
+              this.messageError = this.translateErrorMessage(error.message)
+              break
+            default:
+              this.messageError = this.translateErrorMessage(error.message)
+              break
+          }
+          // Wolle: this.toastError(this.$t('error.email-already-sent'))
+          // Wolle: shall the alert be replaced by a toaster or shall only the page message be shown?
         })
     },
+    // Wolle: remove this?
```

Comment:
For me it would be nice if we have an additional error toast message even when we show the error directly on the page.

This gets more attention of the user in my eyes.
@ogerly @Mogge